### PR TITLE
Add CRUD operations to Books

### PR DESCRIPTION
**Pull request by marcegl91**
## Summary
- Add CRUD operations to Books
## Trello Card
https://trello.com/c/v89X0OZR/18-crear-las-operaciones-crud-de-book

### CODE REVIEW

File: src/main/java/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -1,18 +1,116 @@
...
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import wolox.training.exceptions.BookIdMismatchException;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
 
-@Controller
+/**
+ * BookController.java Controller for Books
```
Comment (leandrocostawolox) [inline src/main/java/wolox/training/controllers/BookController.java]: Same as before, the class name is not necessary

File: src/main/java/wolox/training/controllers/BookController.java [hunk 2]
```diff
@@ -1,18 +1,116 @@
...
     @GetMapping("/greeting")
     public String greeting(
         @RequestParam(name = "name", required = false, defaultValue = "World") String name,
         Model model) {
         model.addAttribute("name", name);
         return "greeting";
     }
+
+    /**
+     * Find all books.
```
Comment (leandrocostawolox) [inline src/main/java/wolox/training/controllers/BookController.java]: You can use here something like this:
```
    /**
     * Find all {@link Book}s.
```

So there will be a link to the Book class :) 
Same at the rest of the Javadocs

**Review by leandrocostawolox — CHANGES_REQUESTED** [review]
Great! I wrote some comments :)

**Review by zeminlu — APPROVED** [review]
Looks good!

**Pull request merged**